addons/mail/static/src/models/composer/composer.js

```diff
@@ -376,6 +376,10 @@ function factory(dependencies) {
             if (!this.thread) {
                 return;
             }
+            const lastInputChar = this.textInputContent.substring(this.textInputCursorStart - 1, this.textInputCursorStart);
+            if ((this.suggestionDelimiter || lastInputChar) === '/') {
+                return;
+            }
```

Comment:
Hello, please use `_getCommandFromText(this.textInputContent)` as suggested before.
Also don't forget to add a test to cover this case.